tests/ClientTest.php

```diff
@@ -54,6 +54,14 @@ public function provideInvalidConstructorDriverOptions()
 
         $options[][] = ['autoEncryption' => ['keyVaultClient' => 'foo']];
 
+        foreach ($this->getInvalidStringValues() as $value) {
+            $options[][] = ['driver' => ['name' => $value]];
+        }
+
+        foreach ($this->getInvalidStringValues() as $value) {
+            $options[][] = ['driver' => ['version' => $value]];
```

Comment:
Should we also test `platform` here?

Reply:
Added, but requires a little extra work since we only pass platform information through to the driver.

Reply:
Oh, I didn't realize that when I left my first comment. I see now that PHPLIB is only throwing its own `InvalidArgumentException` for the two options _it_ merges. Thanks for adding this just the same.